source/commands_functions/create_command.py

```diff
@@ -0,0 +1,20 @@
+from commands_functions.abstract_command import AbtractCommand
+from commands_functions.schema_keys import Keys
+from models.database import Database
+from outputs.exceptions import *
+from outputs.status import Status
+
+
+
+class CreateCommand(AbtractCommand):
+    def __init__(self, schema_path):
+        self.schema_path = schema_path
+        CreateCommand.__validate(schema_path)
+    @staticmethod  
+    def __validate(schema_path):
+        if schema_path == None:
+            raise MissingInput(status = Status.MissingInput, message = "the schema is missing")
+    
+    def execute(self):
+        database = Database(schema_path = self.schema_path)
```

Comment:
Database should take a json object. You should convert the path to json object then pass it to Database